client/src/models/enums/contactQuestioningExcelFields.ts

```diff
@@ -25,19 +24,28 @@ export const ContactedPersonFieldMapper: ContactedPersonExcel= {
     doesHaveBackgroundDiseases: 'האם סובל ממחלות רקע',
     isolationAddress: {
         city: 'יישוב השהייה בבידוד',
-        street: 'רחוב השהייה בבידוד',
-        houseNum: 'מספר בית השהייה בבידוד',
-        apartment: 'מספר דירה השהייה בבידוד',
     },
     doesFeelGood: 'האם חש בטוב',
     doesNeedHelpInIsolation: 'האם נדרש סיוע עבור מקום בידוד',
     repeatingOccuranceWithConfirmed: 'מפגש חוזר עם המאומת',
     doesLiveWithConfirmed: 'האם חי באותו הבית עם המאומת',
     doesWorkWithCrowd: 'עבודה עם קהל במסגרת העבודה',
     doesNeedIsolation: 'הקמת דיווח בידוד',
+    cityId: 'מזהה עיר',
+    streetId: 'מזהה רחוב',
+    houseNum: 'מספר בית השהייה בבידוד',
+    apartment: 'מספר דירה השהייה בבידוד',
```

Comment:
Export the strings to const due to reusing them

Reply:
one of the uses is in a type definition and the other use is in an object definition  , how would you suggest doing that?

Reply:
re-fixed in [here ](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1073/commits/c965e23f16515b48c43befaddf05cf803a1a50f4), now the format is string instead of the actual value in the type field , makes more sense this way - the value can now be changed without depending on the type to change as well